kogito-infinispan-persistence-springboot/src/main/resources/org/acme/deals/SubmitDeal.bpmn2

```diff
@@ -1,125 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Deleted this bpmn but not added the new ones ?

Reply:
Hi @radtriste great catch, added now